src/frontend/src/pages/index.js

```diff
@@ -1,23 +1,48 @@
-import React from 'react';
-import { StaticQuery, graphql } from 'gatsby';
-
-import Layout from '../components/Layout/Layout';
+import React, { useEffect, useState } from 'react';
+import Header from '../components/Header';
+import Banner from '../components/Banner';
+import Posts from '../components/Posts';
+import ScrollToTop from '../components/ScrollToTop';
+import SEO from '../components/SEO';
+import useSiteMetaData from '../hooks/use-site-metadata';
 import './index.css';
 
 export default function IndexPage() {
+  const [numPages, setNumPages] = useState(1);
+  const [posts, setPosts] = useState([]);
+  const { telescopeUrl } = useSiteMetaData();
+
+  useEffect(() => {
+    async function getPosts(pageNum = 1) {
+      let postsData = [];
+      try {
+        const res = await fetch(`${telescopeUrl}/posts?page=${pageNum}`);
+        const postsUrls = await res.json();
+        postsData = await Promise.all(
+          postsUrls.map(async ({ url }) => {
+            const tmp = await fetch(`${telescopeUrl}${url}`);
```

Comment:
Error handling for non-200 here too.

Reply:
Checking for `tmp.ok` now, will throw error if there's an issue.